frontend/public/components/dashboards-page/overview-dashboard/events-card.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { DashboardCard, DashboardCardHeader, DashboardCardBody, DashboardCardTitle } from '../../dashboard/dashboard-card';
```

Comment:
Current convention is to separate external imports from console imports with a blank line

Reply:
blank line added